sql/core/src/test/resources/sql-tests/results/cast.sql.out

```diff
@@ -286,9 +286,15 @@ struct<CAST(interval 3 month 1 hour AS INTERVAL):interval>
 -- !query
 SELECT CAST(interval 3 month 1 hour AS string)
 -- !query schema
-struct<CAST(INTERVAL '3 months 1 hours' AS STRING):string>
```

Comment:
The intention of this test is not to check mixed fields. How about we update the test to use year-month and day-time interval literals?